tests/utils.go

```diff
@@ -20,7 +20,13 @@
 package tests
 
```

Comment:
Please change this file name to something like :pod_utils so it will be utils for pods only and not helpers junkyard :D

Reply:
This file has 2 functions: 1 performs ssh to node, and the other exec to pod.
Is that alright if I split them to `pod_utils.go` and `node_utils.go`?

Reply:
Done